### DISCUSSION TITLE
Visual artifacts when path layers overlap

### DISCUSSION THREAD

**Discussion by Shade5** [visgl/deck.gl#6168] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/13672946/132245097-ee314857-dfca-496d-83c8-de23508c5b39.png)

Issue https://github.com/visgl/deck.gl/issues/6166

I'm seeing artifacts with the PathLayer, I tried setting `depthTest=False`, but that did not help either.

Has anyone else seen the issue? I've added code to replicate this in the issue above

**Comment by Shade5** [comment, marked as answer]
Actually I fixed it by setting a random height for each path. This way there is no rounding error when they overlap. The probelm was that `z=0` for all the paths
![image](https://user-images.githubusercontent.com/13672946/132246093-0dfe77d0-861a-46fe-8313-837c8f0e5d92.png)

**Comment by nashworks** [comment]
if you dont mind me asking - i do get the same issue as we speak - overlapping pathlayers that bleed one into another - and obvious z-fight - but even adding the code according to your post - i still get the same issue with artifcats, jagged edges, and bleeding layers - 

here is what i have:

`let aActualRouteLineDeckLayer = new deck.PathLayer({id:'aActualRouteLineDeckLayer', data:decodePolylineArrayDeck, getPath:d => d.path, getWidth:d => 7, getColor:[0,176,255,255], capRounded:true, jointRounded:true, widthMinPixels:2, widthUnits:'pixels', parameters:{getPolygonOffset:[0,-44], fp64:true, depthTest:true}});

aActualRouteLineDeckOverlay = new deck.GoogleMapsOverlay({layers:[aActualRouteLineDeckLayer]});

aActualRouteLineDeckOverlay.setMap(globalAppMap);

let aActualRouteLineBorderDeckLayer = new deck.PathLayer({id:'aActualRouteLineBorderDeckLayer', data:decodePolylineArrayDeck, getPath:d => d.path, getWidth:d => 9, getColor:[25,103,210,255], capRounded:true, jointRounded:true, widthMinPixels:2, widthUnits:'pixels', parameters:{getPolygonOffset:[0,-33], fp64:true, depthTest:true}});

aActualRouteLineBorderDeckOverlay = new deck.GoogleMapsOverlay({layers:[aActualRouteLineBorderDeckLayer]});

aActualRouteLineBorderDeckOverlay.setMap(globalAppMap);
`
